guides/source/active_record_validations.md

```diff
@@ -1052,6 +1093,8 @@ class Person < ApplicationRecord
 end
 ```
 
+To translate error messages, see the [I18n guide](i18n.html#error-message-scopes).
```

Comment:
Comment for the "allow_blank" section irb code block, line 1047 (Sorry Github won't allow tagging there as it didn't change). Would it make sense to add a line that returns `false` from `valid?` for contrast the examples that return `true`?

Reply:
✅